src/block/accordion/index.php

```diff
@@ -66,3 +66,76 @@ function stackable_load_accordion_frontend_polyfill_script() {
 	}
 	add_action( 'stackable/accordion/enqueue_scripts', 'stackable_load_accordion_frontend_polyfill_script' );
 }
+
+
+if ( ! class_exists( 'Stackable_Accordion_FAQ_Schema') ) {
+	class Stackable_Accordion_FAQ_Schema {
+		public static $accordion_faq_schema = [];
+
+		function __construct() {
+			add_filter( 'render_block_stackable/accordion', array( $this, 'stackable_render_block_accordion_FAQ_schema' ), 10, 2 );
+		}
+
+		public static function stackable_accordion_get_faq_schema() {
+			echo '<script type="application/ld+json">' . implode( ', ', self::$accordion_faq_schema ) . '</script>';
+		}
+
+		public static function stackable_add_schema_to_footer() {
+			add_filter( 'wp_footer', array( 'Stackable_Accordion_FAQ_Schema', 'stackable_accordion_get_faq_schema' ) );
+		}
```

Comment:
Looks like this is called by `self::stackable_add_schema_to_footer();` multiple times - while this should only be done once. Instead of doing this, just add the filter for `wp_footer` in the `__construct()`. Something like:

```php
function __construct() {
	add_filter( 'wp_footer', array( $this, 'print_faq_schema' ) );
}

public function print_faq_schema() {
	if ( count( $this->faq_answers ) ) {
		// echo...
	}
}
```